include/CppUTest/CommandLineTestRunner.h

```diff
@@ -32,6 +32,7 @@
 #include "TestOutput.h"
 #include "CommandLineArguments.h"
 #include "TestFilter.h"
+#include <cwchar>
```

Comment:
Yeah, though we are going to have problems with wchar_t not being a type without #include...